src/Config.cpp

```diff
@@ -43,6 +45,87 @@ Config::Config() {
 
     node_["double_quote_to_single_quote"] = false;
     node_["single_quote_to_double_quote"] = false;
+
+    // Validators
+    auto validate_integer = [](std::string key, std::any elem) {
+        int value = std::any_cast<int>(elem);
+        if (value > 0) {
```

Comment:
I would split this like you did because it might be desirable to have some configurations with 0s as well.
```suggestion
        if (value >= 0) {
```
And then make `validators["spaces_before_call"] = validate_integer;` as well